### ISSUE TITLE
FE | Profile | 508-defect-4 [DESCRIPTIVE TEXT] Profile - Notification Settings: No announcement of the type of telephone number (mobile) being read aloud

### ISSUE THREAD

**Issue by TiffanyPender** [department-of-veterans-affairs/va.gov-team#46609]
### Point of contact

@TiffanyPender

### Severity level

4, Minor. Consider fixing or exploring in 2-4 sprints post-launch.

### Details

As a screen reader user, it is important that content is properly labeled so that I can understand the type of information that is being presented.

Within Profile Notification Settings, near the top of the content, the Veteran’s mobile number is listed, followed by a link to update their mobile phone number. The text content listed beforehand does not truly indicate what type of phone number is being read aloud by screen readers. Users may be able to guess that this is their mobile number, but the lack of specific callout text may create confusion.

### Reproduction steps

1. In staging, login to a LOA3 user's account.
2. Navigate to [Profile](https://staging.va.gov/profile) - Notification Settings
3. With a screen reader, after "We'll use the contact information from your profile to send you the notifications you choose," verify that there is no text read aloud that indicates a mobile number is about to be read.

*Note: add "mobile number" as a screen-reader only span before the phone number

### Proposed solution or next steps

Add in additional text before the listed mobile number to indicate that this is the Veteran’s mobile number on file. This information would be beneficial to all users, not just screen reader users.

### References, articles, or WCAG support

[1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships)
[4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/quickref/#name-role-value)

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [X] Something else

**Comment by SKasimow** [comment]
Need to open a design ticket first

**Comment by SKasimow** [comment]
Closing this...new FE ticket opened #49885
